packages/contract-wrappers/src/index.ts

```diff
@@ -11,50 +48,61 @@ export {
     ExchangeCancelUpToEventArgs,
     ExchangeAssetProxyRegisteredEventArgs,
     ExchangeContract,
-    DevUtilsContract,
-    ForwarderContract,
-    CoordinatorContract,
-    CoordinatorRegistryEventArgs,
-    CoordinatorRegistryEvents,
-    CoordinatorRegistryCoordinatorEndpointSetEventArgs,
-    CoordinatorRegistryContract,
-    IValidatorContract,
-    IWalletContract,
+    ExchangeProtocolFeeCollectorAddressEventArgs,
+    ExchangeProtocolFeeMultiplierEventArgs,
+    ExchangeTransactionExecutionEventArgs,
+} from './generated-wrappers/exchange';
+export { ForwarderContract } from './generated-wrappers/forwarder';
+export { IValidatorContract } from './generated-wrappers/i_validator';
+export { IWalletContract } from './generated-wrappers/i_wallet';
```

Comment:
Let's remove IWalletContract, IValidatorContract, most devs don't use those. They _can_ get them from `contracts-*` if need be.

Reply:
removed!